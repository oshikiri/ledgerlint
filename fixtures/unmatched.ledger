2020-03-26* toilet paper
