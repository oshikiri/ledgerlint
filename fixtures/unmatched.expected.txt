fixtures/unmatched.ledger:1 Header unmatched
